src/components/header/view/headerView.tsx

```diff
@@ -114,9 +114,9 @@ const HeaderView = ({
             <IconButton
```

Comment:
prop that enables qr button has misleading meaning... check here, https://github.com/ecency/ecency-mobile/blob/ff9d53266c85db66ce84c17209d519b0ed1f7568/src/components/header/view/headerView.tsx#L113